examples/advanced/variables.tf

```diff
@@ -7,6 +7,7 @@ variable "ibmcloud_api_key" {
 variable "prefix" {
   type        = string
   description = "Prefix name for all related resources"
+  default     = "andrej-t"
```

Comment:
default value was purposely removed from examples - please remove it. See https://github.ibm.com/GoldenEye/issues/issues/5505